extractor/filesystem/filesystem.go

```diff
@@ -173,11 +173,11 @@ func runOnScanRoot(ctx context.Context, config *Config, scanRoot *scalibrfs.Scan
 // are expected to be relative to the scan root.
 // This function is exported for TESTS ONLY.
 func InitWalkContext(ctx context.Context, config *Config, absScanRoots []*scalibrfs.ScanRoot) (*walkContext, error) {
-	pathsToExtract, err := stripAllPathPrefixes(config.PathsToExtract, absScanRoots)
+	pathsToExtract, err := stripAllPathPrefixesAndToSlash(config.PathsToExtract, absScanRoots)
```

Comment:
Let's not put too many things into one helper lib - can we have a separate helper function for calling ToSlash on the string array?